### ISSUE TITLE
POV INI File Save/Load Issues

### ISSUE THREAD

**Issue by shro2023** [vpinball/vpinball#1081]
**Describe the bug**
POV INI file is not being saved / loaded correctly. 

**To Reproduce**
1. Open POV Editor from either camera button in VPX Editor or Adjust Camera option when playing a table
2. Make amendments to POV
3. Press Start button to save INI file
4. Open INI file and compare values to those shown in POV Editor.
5. Values stored in INI file for Scale, Offsets etc. do not reflect the same values as those displayed in the POV Editor
6. Close Editor and reload table by either double-clicking table directly or opening Editor and then selecting table
7. Table POV will not reflect camera angle set in step 2.

If the bug appears when playing a specific table, give a link to a place where the exact version of the table file can be downloaded.

**Expected behavior**
1. INI File values to be showing same values as POV Editor values shown on screen when completed editing
2. Camera view to be consistent when loading table with the view shown in the editor when creating / amending the INI file

**Versions & Settings**
 - VPX: 10.8 GL x64 1712
 - Windows 11 Ghost Spectre

**Comment by shro2023** [comment]
Fantastic! Thanks @vbousquet - initial testing seems to suggest this is fully sorted and even better does not require me to reconfigure all my previous version POVs to INIs as I was expecting would be required. Thanks for adding in the backward compatability - very happy!

**Comment by Ltek** [comment]
> This should now be fixed. I believe the problem you faced was linked to situations where you have ini and pov files conflicting (and eventually autopov file could cause conflict as well). I will close for now. Please test and tell me if you're still facing issues with this

I dont think this is fixed. I have no / zero POV files and 10.8 6 & 7 do not load the VPinballX.ini... I've documented this in this ticket... https://github.com/vpinball/vpinball/issues/1212
